packages/react-components/src/components/VideoTile.tsx

```diff
@@ -307,30 +306,30 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
   const canShowLabel = showLabel && (displayName || (showMuteIndicator && isMuted));
   const participantStateString = participantStateStringTrampoline(props, locale);
   return (
-    <Ref innerRef={videoTileRef}>
-      <Stack
-        data-ui-id={ids.videoTile}
-        className={mergeStyles(
-          rootStyles,
-          {
-            background: theme.palette.neutralLighter,
-            borderRadius: theme.effects.roundedCorner4
-          },
-          isSpeaking && {
-            '&::before': {
-              content: `''`,
-              position: 'absolute',
-              zIndex: 1,
-              border: `0.25rem solid ${theme.palette.themePrimary}`,
-              borderRadius: theme.effects.roundedCorner4,
-              width: '100%',
-              height: '100%'
-            }
-          },
-          styles?.root
-        )}
-        {...longPressHandlersTrampoline}
-      >
+    <Stack
+      data-ui-id={ids.videoTile}
+      className={mergeStyles(
+        rootStyles,
+        {
+          background: theme.palette.neutralLighter,
+          borderRadius: theme.effects.roundedCorner4
+        },
+        isSpeaking && {
+          '&::before': {
+            content: `''`,
+            position: 'absolute',
+            zIndex: 1,
+            border: `0.25rem solid ${theme.palette.themePrimary}`,
+            borderRadius: theme.effects.roundedCorner4,
+            width: '100%',
+            height: '100%'
+          }
+        },
+        styles?.root
+      )}
+      {...longPressHandlersTrampoline}
+    >
+      <div ref={videoTileRef} style={{ width: '100%', height: '100%' }}>
```

Comment:
Nice, we should do get rid of dependencies of northstar more and more

Btw, we do we not wrap the Stack element anymore?

Reply:
so the stack that is wrapping this div is what is setting the height for the tile. I moved the div that is setting the ref inside so that the tile renders as normal. For some reason when the div was outside it would break the video tile and it wouldn't have a height being passed to the reference.

Reply:
nit: put style { width: '100%', height: '100%' } in VideoTile.styles.ts 😅

Reply:
I guess Ref doesn't create any new element but div does? And that could be the reason, LGTM if this would not create any regressions, but move the style tag into a separated file (./styles/VideoTile.styles)